### DISCUSSION TITLE
do not work canvas

### DISCUSSION THREAD

**Discussion by faidems8** [BenJeau/react-native-draw#60] [Q&A]
i installed 

npm install @benjeau/react-native-draw
expo install react-native-svg
npm install @benjeau/react-native-draw-extras

and wrote this code but got an error

![image](https://user-images.githubusercontent.com/99948363/165035358-cda82af0-f13a-4787-9940-6073fbe02e2e.png)

![image](https://user-images.githubusercontent.com/99948363/165034927-2ac4e83c-d72d-4e0e-8372-0e3317db29b7.png)

what's wrong..?? please help

**Comment by ChrisBDuffy** [comment]
try
expo install react-native-gesture-handler

**Reply by reyryan** [reply]
@faidems8 have you manage to find a solution to this issue ? I encountered a similar issue as well when using react-native:0.63.2.

 ERROR    TypeError: undefined is not an object (evaluating '_reactNativeGestureHandler.Gesture.Pan')

----------------------------
package.json (snippet)
```
"dependencies": {
   "@benjeau/react-native-draw": "^0.8.1",
   "react-native-gesture-handler": "^1.7.0",
    ....
    "react": "16.13.1",
    "react-native": "0.63.2",
   "react-native-svg": "^12.3.0"
}

```
screen.js
```
import React from 'react';
import { Canvas } from '@benjeau/react-native-draw';

export const BookEditScreen = () => {
  return (
    <Canvas />
  )
}
```

**Reply by vsalguero** [reply]
I open the file Canvas.tsx in node_modules/@benjeau/react-native-draw/src And I saw a warning in the import from 

```
import {
  Gesture,
  GestureDetector,
  GestureHandlerRootView,
} from 'react-native-gesture-handler';
```

Then I thought it could be due to the package version, I had version 1.10.3 when uninstalling the package and installing it again updated to version 2.13.4 and the problem was solved.

```
npm uninstall react-native-gesture-handler --save

npm install react-native-gesture-handler --save
```
